lib/google-auth.js

```diff
@@ -0,0 +1,33 @@
+import google from 'googleapis';
+import util from './util';
+
+let KEY;
```

Comment:
When I first read this `KEY` var, I thought it would be *a* key itself, but later in the code I noticed it is actually an object holding different things (a key, an email, ...). I think it deserves a more descriptive name :)